src/client/index.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import { render } from 'react-dom'
-import 'bootstrap/dist/css/bootstrap.css'
-
+import 'semantic-ui-css/semantic.min.css'
```

Comment:
@siawo Is there any way to selectively choose what CSS styles we introduce in our project via SemanticUI?

Reply:
Yes, there are ways to selectively build the css files with components being used in the project.

Reply:
Will create the custom build at the end after getting list of all the required css for the project.

Reply:
Okay. Tracking the custom build via issue #11